t/api/08-jobtemplates.t

```diff
@@ -516,15 +517,30 @@ is_deeply(['opensuse', 'opensuse test'], [sort keys %$yaml], 'YAML of all groups
 $t->get_ok('/api/v1/job_templates_scheduling/' . $opensuse->id)->status_is(200);
 $yaml = $t->tx->res->body;
 is($t->tx->res->body, $yaml, 'No document start marker by default');
-$yaml = load_yaml(string => $t->tx->res->body);
+$yaml = load_yaml(string => $t->tx->res->json);
```

Comment:
This looks very odd to me... surely we are **not** getting JSON here?

Reply:
Same question applies to all other instances later in this file.

Reply:
The client in the test by default requests JSON. What we are getting back is a JSON string containing YAML. `$res->json` returns this string as a perl string already decoded.
Then we load it with `load_yaml`.

Reply:
Ah, nevermind then, so that's what I got confused with before...